content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## User facing changes
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis).
```

Comment:
Can we link here instead: https://kubernetes-sigs.github.io/service-apis/

Reply:
Actually, those look pretty sparse too if you start clicking on links. Either way, that GitHub has very little content. Is there somewhere better to link?

Reply:
No that is pretty much the extent of it so far

Reply:
Sorry this says user facing but it is not clear how it is related to istio user?   If there is a link to guide user how to leverage this it would be great

Reply:
This is the PR that implemented the work: https://github.com/istio/istio/pull/21594/. I don't believe there is any documentation other than generated docs.